### ISSUE TITLE
Gradle verification

### ISSUE THREAD

**Issue by paul-dingemans** [pinterest/ktlint#1794]
I have checked out a fresh copy of the ktlint project. Next I run a full build and all tests passes successfully. However, when I next reload all Gradle projects, I get errors below:
```
Dependency verification failed for org.codehaus.groovy:groovy:3.0.13
4 artifacts failed verification:
  - gradle-7.6-src.zip (gradle:gradle:7.6) from repository Gradle distributions
  - gradle-kotlin-dsl-plugins-2.4.1-sources.jar (org.gradle.kotlin:gradle-kotlin-dsl-plugins:2.4.1) from repository Gradle Central Plugin Repository
  - groovy-3.0.13-sources.jar (org.codehaus.groovy:groovy:3.0.13) from repository Gradle Libs
  - groovy-3.0.13.pom (org.codehaus.groovy:groovy:3.0.13) from repository Gradle Libs
If the artifacts are trustworthy, you will need to update the gradle/verification-metadata.xml file by following the instructions at https://docs.gradle.org/7.6/userguide/dependency_verification.html#sec:troubleshooting-verification

Open this report for more details: file:///Users/pdingemans/git/ktlint/ktlint/build/reports/dependency-verification/at-1674839811361/dependency-verification-report.html
```

Running command `./gradlew -q dependencies` shows following dependency hierarchy:
```

------------------------------------------------------------
Root project 'ktlint-root'
------------------------------------------------------------

ktlint
\--- project :ktlint-cli
     +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0
     |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.8.0
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.8.0
     |    |    \--- org.jetbrains:annotations:13.0
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.8.0
     |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.8.0 (*)
     +--- project :ktlint-core
     |    +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.8.0
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.8.0 (*)
     |    |    +--- org.jetbrains.kotlin:kotlin-script-runtime:1.8.0
     |    |    +--- org.jetbrains.kotlin:kotlin-reflect:1.6.10
     |    |    +--- org.jetbrains.kotlin:kotlin-daemon-embeddable:1.8.0
     |    |    +--- org.jetbrains.intellij.deps:trove4j:1.0.20200330
     |    |    \--- net.java.dev.jna:jna:5.6.0
     |    +--- org.ec4j.core:ec4j-core:0.3.0
     |    +--- io.github.microutils:kotlin-logging-jvm:2.1.23
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.6.0 -> 1.8.0
     |    |    +--- org.slf4j:slf4j-api:1.7.32 -> 2.0.0
     |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.0 -> 1.8.0 (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- project :ktlint-cli-reporter-core
     |    +--- project :ktlint-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- project :ktlint-cli-reporter-plain
     |    +--- project :ktlint-cli-reporter-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- project :ktlint-ruleset-standard
     |    +--- project :ktlint-ruleset-core
     |    |    +--- project :ktlint-core (*)
     |    |    +--- project :ktlint-rule-engine
     |    |    |    +--- project :ktlint-core (*)
     |    |    |    +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.8.0 (*)
     |    |    |    +--- org.ec4j.core:ec4j-core:0.3.0
     |    |    |    +--- io.github.microutils:kotlin-logging-jvm:2.1.23 (*)
     |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     |    |    +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.8.0 (*)
     |    |    +--- org.ec4j.core:ec4j-core:0.3.0
     |    |    +--- io.github.microutils:kotlin-logging-jvm:2.1.23 (*)
     |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     |    +--- project :ktlint-rule-engine (*)
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     |    +--- project :ktlint-core (*)
     |    \--- io.github.microutils:kotlin-logging-jvm:2.1.23 (*)
     +--- project :ktlint-ruleset-test
     |    +--- project :ktlint-ruleset-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- info.picocli:picocli:4.6.3
     +--- ch.qos.logback:logback-classic:1.3.0
     |    +--- ch.qos.logback:logback-core:1.3.0
     |    \--- org.slf4j:slf4j-api:2.0.0
     +--- project :ktlint-cli-reporter-baseline
     |    +--- project :ktlint-cli-reporter-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- project :ktlint-cli-reporter-checkstyle
     |    +--- project :ktlint-cli-reporter-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- project :ktlint-cli-reporter-json
     |    +--- project :ktlint-cli-reporter-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- project :ktlint-cli-reporter-format
     |    +--- project :ktlint-cli-reporter-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- project :ktlint-cli-reporter-html
     |    +--- project :ktlint-cli-reporter-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     +--- project :ktlint-cli-reporter-plain-summary
     |    +--- project :ktlint-cli-reporter-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
     \--- project :ktlint-cli-reporter-sarif
          +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0 (*)
          +--- project :ktlint-cli-reporter-core (*)
          \--- io.github.detekt.sarif4k:sarif4k:0.2.0
               +--- org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1
               |    \--- org.jetbrains.kotlinx:kotlinx-serialization-json-jvm:1.4.1
               |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.20 -> 1.8.0 (*)
               |         +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.20 -> 1.8.0
               |         +--- org.jetbrains.kotlinx:kotlinx-serialization-core:1.4.1
               |         |    \--- org.jetbrains.kotlinx:kotlinx-serialization-core-jvm:1.4.1
               |         |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.20 -> 1.8.0 (*)
               |         |         +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.20 -> 1.8.0
               |         |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.20 -> 1.8.0 (*)
               |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.20 -> 1.8.0 (*)
               \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.20 -> 1.8.0 (*)

(*) - dependencies omitted (listed previously)
```
It does not contain the `groovy` dependency.

I have tried running `./gradlew --write-verification-metadata pgp,sha256 `. After running that command, the groovy error still is present:
```
Dependency verification failed for org.codehaus.groovy:groovy:3.0.13
2 artifacts failed verification:
  - groovy-3.0.13-sources.jar (org.codehaus.groovy:groovy:3.0.13) from repository Gradle Libs
  - groovy-3.0.13.pom (org.codehaus.groovy:groovy:3.0.13) from repository Gradle Libs
If the artifacts are trustworthy, you will need to update the gradle/verification-metadata.xml file by following the instructions at https://docs.gradle.org/7.6/userguide/dependency_verification.html#sec:troubleshooting-verification

Open this report for more details: file:///Users/pdingemans/git/ktlint/ktlint/build/reports/dependency-verification/at-1674840145583/dependency-verification-report.html
```

What can I do to get rid of it?

**Comment by paul-dingemans** [comment]
@Goooler Can you please help me out on this problem?
